Multisig/UI/App/NavigationRouter.swift

```diff
@@ -39,6 +39,36 @@ class DefaultNavigationRouter: NavigationRouter {
     func navigate(to route: NavigationRoute) {
         sceneDelegate?.navigate(to: route)
     }
+
+    private let pattern = "^https://gnosis-safe.io/app/([-a-zA-Z0-9]{1,20}):(0x[a-fA-F0-9]{40})/([-a-zA-Z0-9]{1,20})(/[-a-zA-Z0-9_]{1,118})?"
```

Comment:
just a note: this will break when the tx id format will change (the last term)

Reply:
Will it changed? 
any suggestion to get ride of this ?

Reply:
thinking about this... perhaps not.

good to go for  now.